File: c/c99/1 - Iniciante/1036_formula_de_bhaskara.c
#include <stdio.h>
#include <math.h>

int main() {
    double a, b, c, delta, x1, x2;
    scanf("%lf %lf %lf", &a, &b, &c);
    if (a == 0){
        printf("Impossivel calcular\n");
    }
    else {
        delta = pow(b, 2) - 4 * a * c;
        if (delta >= 0) {
            x1 = (- b + pow(delta, 0.5)) / (2 * a);
            x2 = (- b - pow(delta, 0.5)) / (2 * a);
            printf("R1 = %.5lf\nR2 = %.5lf\n", x1, x2);
        }
        else {
            printf("Impossivel calcular\n");
        }
    }
}

File: python/1 - Iniciante/1036_formula_de_bhaskara.py
import math

A, B, C = map(float, input().split())

Δ = (B**2)-(4*A*C)

if Δ < 0:
    print(f'Impossivel calcular')
elif A <= 0:
    print(f'Impossivel calcular')
else:
    R1 = (-B +math.sqrt(Δ)) / (2*A)
    R2 = (-B -math.sqrt(Δ)) / (2*A)
    print(f'R1 = {R1:.5f}')
    print(f'R2 = {R2:.5f}')
